src/server/pps/server/gc.go

```diff
@@ -57,16 +57,17 @@ func (a *apiServer) runGC(ctx context.Context, pipelineInfo *ppsclient.PipelineI
 			continue
 		}
 		for _, jobID := range jobIDs.Jobs {
-			zero := int64(0)
-			falseVal := false
-			go func(jobID string) {
-				if err := a.kubeClient.Extensions().Jobs(a.namespace).Delete(jobID, &api.DeleteOptions{
-					GracePeriodSeconds: &zero,
-					OrphanDependents:   &falseVal,
-				}); err != nil {
-					// TODO: if the error indicates that the job has already been
-					// deleted, just proceed to the next step
-					log.Errorf("error deleting kubernetes job %s: %s", jobID, err)
+			jobID := jobID
+			go func() {
+				jobInfo, err := client.InspectJob(ctx, &ppsclient.InspectJobRequest{
+					Job: &ppsclient.Job{ID: jobID},
+				})
+				if err != nil {
+					log.Errorf("error deleting job: %s", err)
+					return
+				}
+				if err := a.deleteJob(ctx, jobInfo); err != nil {
+					log.Errorf("error deleting job: %s", err)
```

Comment:
Was this part of the changes? It looks like it might have gotten included from a merge or something weird. I think I made this change about a week ago to fix the jobs not getting removed correctly. I'm a little unsure why it got included in the diff though since the new version of the code seems to be what's in master. We should check on this before we merge just so we don't get bitten by any weird merge issues.